.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
```

Comment:
- Optional 
- The name for workflow runs 
  - generated from the workflow
  - appear in the list of workflow runs on your repository's "Actions" tab

- `${{ github.actor }}`
  - an expression with the github context 
    - to display the username of the actor that triggered the workflow run
  - For more information, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name)."